client/jetpack-connect/style.scss

```diff
@@ -777,3 +776,167 @@ $colophon-height: 50px;
 .jetpack-connect__connect-button {
 	text-align: center;
 }
+
+/**
+ * Onboarding styles
+ **/
+.is-section-jetpack-connect:not(.is-add-site-page),
+.is-jetpack-login {
+	background-color: var( --color-jetpack-onboarding-background );
+
+	&.layout {
+		position: relative;
+		min-height: calc( 100% - #{$colophon-height} );
+		padding-bottom: $colophon-height;
+	}
+
+	.login__form-header,
+	.formatted-header {
+		color: var( --color-jetpack-onboarding-text );
+	}
+
+	.jetpack-logo__text,
+	.wpcom-colophon path {
+		fill: var( --color-jetpack-onboarding-text );
+	}
+
+	.formatted-header__subtitle,
+	.payment-methods,
+	.jetpack-connect__help-button {
+		color: var( --color-primary-100 );
+	}
+
+	.card,
+	.site-type__wrapper .card,
+	.site-topic__wrapper .card {
+		box-shadow: none;
+		border-radius: 3px;
```

Comment:
Hm, this will cause the authorization page that displays a `Site` card to look a little odd with the border radius in between:

![](https://cldup.com/A_zeHKnxld-2000x2000.png)

Reply:
That was the intention, actually, but I'm happy to give it more thought. Would you suggest merging the cards or space them more apart?

Reply:
Oh, if that was intentional, let's go with it. I'm just too used with that being a border instead.

Reply:
Hopefully it's an intermediate styling that starts to approximate the simplified sign up/log in changes.